# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../wings3d/wings-1.4.1-wx-gl.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- wings-1.4.1/plugins_src/primitives/wpc_image.erl.orig	2012-05-13 01:57:30.060837481 +0200
+++ wings-1.4.1/plugins_src/primitives/wpc_image.erl	2012-05-13 01:57:45.270643901 +0200
@@ -14,7 +14,7 @@
 -module(wpc_image).
 -export([init/0,menu/2,command/2]).
 
--include_lib("esdl/include/gl.hrl").
+-include_lib("wx/include/gl.hrl").
 -include("e3d.hrl").
 -include("e3d_image.hrl").
 -include("wings_intl.hrl").
